phpunit/block-supports/background-test.php

```diff
@@ -242,13 +242,13 @@ public function data_get_background_support_styles() {
 						'source' => 'file',
 					),
 				),
-				'expected_css'     => "background-image:url('https://example.com/image.jpg');background-size:cover;",
+				'expected_css'     => "background-image:url('https://example.com/image.jpg');",
```

Comment:
Just double-checking, it's safe to remove these parts of the tests, because we're already testing the default behaviour (`cover` and `background-position:center` further up in the file in `data_background_block_support`)?

Reply:
That's right. These tests test `gutenberg_get_background_support_styles()` only.

`test_background_block_support()` takes care of the block supports. 

Now that I'm looking again, the tests aren't really worth much as it's just testing the style engine now, which has tests 😄